packages/react/package.json

```diff
@@ -46,7 +46,9 @@
     "@mui/utils": "^5.10.16",
     "@oxygen-ui/primitives": "*",
     "@oxygen-ui/react-icons": "*",
-    "clsx": "^1.2.1"
+    "@storybook/addon-essentials": "^6.5.16",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
Fixed with - https://github.com/wso2/oxygen-ui/pull/66